src/parse.c

```diff
@@ -4643,8 +4643,9 @@ static void p_interface_constant_declaration(tree_t parent, tree_kind_t kind)
       add_interface(parent, d, kind);
       sem_check(d, nametab);
 
-      if (kind == T_GENERIC_DECL && standard() >= STD_08) {
+      if ((standard() >= STD_19) || (kind == T_GENERIC_DECL && standard() >= STD_08)) {
          // Generics are immediately visible in VHDL-2008
+         // Everything is immediately visible in VHDL-2019
          insert_name(nametab, d, NULL);
```

Comment:
Although it's slightly redundant, I think it's clearer split over two cases:

```suggestion
      if (kind == T_GENERIC_DECL && standard() >= STD_08) {
         // Generics are immediately visible in VHDL-2008
         insert_name(nametab, d, NULL);
      }
      else if (standard() >= STD_19) {
         // Everything is immediately visible in VHDL-2019
         insert_name(nametab, d, NULL);
```